colossalai/nn/optimizer/cpu_adam.py

```diff
@@ -1,9 +1,51 @@
 import math
-
 import torch
 
+from colossalai.registry import OPTIMIZERS
+
 
+@OPTIMIZERS.register_module
 class CPUAdam(torch.optim.Optimizer):
+    """Implements Adam algorithm.
+
+    Supports both GPU and CPU. Requires ColossalAI to be installed via
```

Comment:
supports (what) both resident on CPU and GPU?

Reply:
I will make it more detailed.